mathics/builtin/assignment.py

```diff
@@ -981,20 +981,29 @@ class Quit(Builtin):
 
     >> a = 3
      = 3
-    >> Quit[]
+    >> Quit[];
+    >> a
+     = a
+
+    >> a = 3
+     = 3
+    >> Quit;
     >> a
      = a
 
     'Quit' even removes the definitions of protected and locked symbols:
     >> x = 5;
     >> Attributes[x] = {Locked, Protected};
-    >> Quit[]
+    >> Quit[];
```

Comment:
No, of course... It is just that I use to end lines with semicolons when I do not look for output. I think that in this context would be better to do not use it, in order to see all the output.